1136 - Another set of cleanup (#1543)
infor-design/enterprise: src/components/notification/readme.md

---
title: Notification

> description: The notification banner is a non dismissible banner that sits at the top of the page. It has alert colors and a optional link and dismiss button.

Reviewer: ```suggestion
description: The notification banner is a non-dismissible banner that sits at the top of the page. It has alert colors, an optional link, and a dismiss button.
```

demo:
  embedded:
  - name: Showing a notification
    slug: example-index
  - name: No link
    slug: example-no-link
---

## Code Example

> The notification component is a component that inserts a banner at the top of the page for showing user dismissible alerts and messages. Its contents will be removed when the banner is closed. You have control over the message contents and if you would like a link. The message is triggered by selecting the parent content (usually `<body>`) and executing the `notification()` function with various options to create the notification banner.

Reviewer: ```suggestion
The notification component is a component that inserts a banner at the top of the page for showing user-dismissible alerts and messages. Its contents will be removed when the banner is closed. You have control over the message contents including the option for a link. The message is triggered by selecting the parent content (usually `<body>`) and executing the `notification()` function with various options to create the notification banner.
```
